### PR TITLE
Issue #6: Convert views/views_plugin_argument_default_bboxquery.inc to D...

### PR DESCRIPTION
**Pull request by dmurphy1**
Hey @kostajh, this pull request begins the process of converting BBoxQuery to Drupal 8 by renaming/moving the file, updating the base class, and modifying the methods based on the new views QueryParameter plugin class. This is a work in progress and further updates are required, especially to the getArgument method. RE: Issue #6

### CODE REVIEW

**Comment by dmurphy1** [comment]
Actually, just noticed a mistake in my namespacing @kostajh, let me fix that.

File: views/views_plugin_argument_default_bboxquery.inc [hunk 1]
```diff
@@ -4,6 +4,8 @@
  * Contains the BBOX query string argument default plugin.
  */
 
+// @TODO: Delete this file since it's being replaced by src/Plugin/views/argument_default/BBoxQuery.php?
```
Comment (kostajh) [inline views/views_plugin_argument_default_bboxquery.inc]: @dmurphy1 yes we can delete this file. (eventually everything in `/views` will be removed.)

**Comment by kostajh** [comment]
Overall it looks good to me @dmurphy1, the problem is that this was integrated with OpenLayers which doesn't have a D8 port yet, so I'm not sure how we can test it out.

**Comment by dmurphy1** [comment]
Ahh, I see @kostajh - I did not realize that :(

**Comment by kostajh** [comment]
@dmurphy1 it's OK, good to have this done so it's ready whenever OpenLayers is!

**Comment by timstallmann** [comment]
@kostajh @dmurphy1 we should be able to test this by setting up a view with the BBox contextual query and passing in different lat/lng combinations to see what comes out.

**Comment by kostajh** [comment]
@timstallmann cool, let's keep track of this in #8 as well.

**Comment by dmurphy1** [comment]
Thanks for pointing that out @timstallmann!

**Pull request merged**